### PR TITLE
pr for microverse tse approval: milestone 4 category filter

### PR DESCRIPTION
**Pull request by firmoholanda**
in this pr, we implemented the folowing:

- create the CHANGE_FILTER action in actions/index.js;
- create a filter.js;
- implement the CategoryFilter component;
- render the CategoryFilter component in the BooksList component;
- implement the handleFilterChange;
- implement a conditional method which decides which books should be shown in the table;

### CODE REVIEW

File: src/components/CategoryFilter.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const CategoryFilter = ({ handleFilterChange }) => {
+  // const handleFilterChange = (event) => {
```
Comment (Forison) [inline src/components/CategoryFilter.js:5]: - Kindly remove the commented code

**Review by Forison — CHANGES_REQUESTED** [review]
## Status:: Change Required❌.

# 🙋‍♂️ Holanda & Kiprop. 

#### You have done a great job on this one. All the requirements were almost met, however, there is an issue to address🤗.

* [ ] It’s better to delete commented piece of codes. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

### That is it, kindly make the change and submit a re-review😎.

#### Happy coding👨‍💻

Addo Forison

File: src/components/CategoryFilter.js (hunk 1 above)
Comment (firmoholanda) [reply src/components/CategoryFilter.js:5]: we have removed the comment. thanks!

**Review by shloch — APPROVED** [review]
## Hello @firmoholanda, @Cyrus-Kiprop 

accomplished

We're ready for next milestone....THUMBS UP

YOu can merge this

Happy coding

**Pull request merged**